main.go

```diff
@@ -385,6 +395,43 @@ func runPreReleaseScripts(preReleaseScriptsDirpath string, releaseVersion semver
 	return nil
 }
 
+func updateChangelog(changelogFilepath string, releaseVersion string) error {
+	clFile, err := os.ReadFile(changelogFilepath)
+	if err != nil {
+		logrus.Errorf("Error attempting to open changelog file at provided path. Are you sure '%s' exists?", changelogFilepath, err)
+	}
+	tbdRegex, err := regexp.Compile(TBD_VERSION_HEADER_REGEX)
+	if err != nil {
+		logrus.Errorf("Could not parse regexp: '%s'", TBD_VERSION_HEADER_REGEX, err)
+	}
+
+	lines := bytes.Split(clFile, []byte("\n"))
+	for i, line := range lines {
+		// Change current TBD header to Release Version header
+		if tbdRegex.Match(line){
+			releaseVersionHeader := fmt.Sprintf("# %s", releaseVersion)
+			lines[i] = []byte(releaseVersionHeader)
+			break
+		}	
+	}
+	// Add a new TBD head for next release
```

Comment:
is there an easy way to just `prepend` in golang? then this could be reduced to one line.

Reply:
also can do this in one pass instead of two